community-stabilization-fund-api/src/modules/checklists/styles/checklists.module.css

```diff
@@ -2,24 +2,80 @@
   @page {
     size: letter;
     /* you can also specify margins here: */
+    margin: 0;
+    padding: 0;
+    width: 100%;
+    height: 100%;
     /* margin: 25mm; */
     /* margin-right: 45mm;  for compatibility with both A4 and Letter */
   }
+  /* @page {
+    size: 58.7mm 101.6mm;
+    margin: 0 !important;
+    padding: 0 !important;
+  } */
+  .item_checklist_dymo_wrapper {
+    padding: 0 !important;
+    margin: 0 !important;
+    justify-content: flex-start !important;
+    /* display: flex;
+    flex-direction: column;
+    margin: 0 auto;
+    align-items: center;
+    height: 100%;
+    padding: 0 !important;
+    width: 1200px !important;
+    justify-content: space-between;
+    align-content: space-between; */
+  }
+
   .dymo_op_one_bag_label {
```

Comment:
So this is what I was talking about when I said styling needs to be separated. There's definitely weird interactions going on in this file if we have to put so many !important's.